pgjdbc/src/main/java/org/postgresql/replication/fluent/AbstractCreateSlotBuilder.java

```diff
@@ -17,4 +28,15 @@ public T withSlotName(String slotName) {
     this.slotName = slotName;
     return self();
   }
+
+  @Override
+  public T withTemporaryOption(boolean temporaryOption) throws SQLFeatureNotSupportedException {
```

Comment:
I don't think there is any need to pass a boolean in to this. Set the default to false and set it to true in the function